protostar/commands/test/environments/fuzz_test_execution_environment.py

```diff
@@ -43,10 +44,24 @@ def is_fuzz_test(function_name: str, state: TestExecutionState) -> bool:
     return bool(params)
 
 
+@dataclass
+class FuzzTestExecutionResult(TestExecutionResult):
+    fuzz_runs_count: int
+
+
 class FuzzTestExecutionEnvironment(TestExecutionEnvironment):
+    @dataclass
+    class FuzzConfig:
+        max_fuzz_examples: int = 100
+
     def __init__(self, state: TestExecutionState):
         super().__init__(state)
         self.initial_state = state
+        self._fuzz_config = FuzzTestExecutionEnvironment.FuzzConfig()
+
+    def set_fuzz_config(self, fuzz_config: FuzzConfig) -> Self:
+        self._fuzz_config = fuzz_config
+        return self
```

Comment:
could the value of this property be passed via constructor? even at the cost of adding extra keyword arguments here and there. I don't like exposing a mutable-like semantics here.

Reply:
It could be passed through constructor however:
- it breaks the factory (TestExecEnv i FuzzTestExecEnv) would accept different argument and the factory
- I try to avoid passing runtime configuration through a constructor because the class can't be instantiated before configuration is known therefore the class can't be injected as a dependency. This forces a developer using Builders as a workaround. In the constructor I tend to pass only dependencies, and I am not sure if the configuration is a dependency. Anyway, in this case it doesn't matter — I'll change it.